src/Spago/RunEnv.hs

```diff
@@ -155,13 +169,14 @@ getConfig = Config.ensureConfig >>= \case
 
 getPurs :: HasLogFunc env => UsePsa -> RIO env PursCmd
 getPurs usePsa = do
-  -- first we decide if we _want_ to use psa, then if we _can_
-  pursCandidate <- case usePsa of
-    NoPsa -> pure "purs"
-    UsePsa -> findExecutable "psa" >>= \case
-      Just _  -> pure "psa"
-      Nothing -> pure "purs"
-  PursCmd <$> findExecutableOrDie pursCandidate
+  purs <- findExecutableOrDie "purs"
+  psa <- case usePsa of
+    NoPsa -> pure Nothing
+    UsePsa -> findExecutable "psa"
+  compilerVersion <- Purs.pursVersion >>= \case
+    Left _ -> die [ "Failed to fetch purs version" ]
```

Comment:
Should we print this error instead of ignoring it?

Reply:
Yeah, good idea, I'll do what's done elsewhere in this file to print it.